### PR TITLE
Adding the ability to set specific version

### LINKED ISSUE #3: Ability to set a specific version number
I think it would be useful to allow setting a specific version number via a parameter. E.g. when you want to update it from a tag.

### PR DESCRIPTION
**Pull request by Piedone**
Fixes #3.

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -20,16 +20,24 @@ inputs:
...
     description: 'The resulting version number'
-    value: ${{ steps.version_increment.outputs.version }}
+    value: ${{ steps.version_set.outputs.version }}
 runs:
   using: "composite"
   steps:
-    - name: Increment VSIX version
-      id: version_increment
+    - name: Set VSIX version
+      id: version_set
```
Comment (timheuer) [inline action.yaml]: I'd like to not change the ID here -- some may be relying on the steps.id output in random places and that may be an unexpected breaking change...

**Review by timheuer — CHANGES_REQUESTED** [review]

File: action.yaml (hunk 1 above)
Comment (Piedone) [reply action.yaml]: Hmm, can you access the output of a step within a composite action you call from the outside? That would be a strange breach of encapsulation. Anyway, I renamed it back.

**Review by timheuer — APPROVED** [review]
Thank you...it may be nothing, but I don't have the depth to know that the workflow of composites can rely on these.  I'm treating anything labeled 'id' as breaking change ;-) -- thanks for the contribution...will merge sometime today (in between meetings)

**Comment by Piedone** [comment]
Sure, and thank you!

**Pull request merged**